packs/equipment-effects/effect-mudrock-snare-failure.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Mudrock Snare (Failure)",
     "system": {
         "description": {
-            "value": "<p>The creature takes a -10-foot circumstance penalty to its Speed for 1 minute or until it Escapes (DC 27).</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Mudrock Snare]</p>\n<p>You take a -10-foot circumstance penalty to your Speed.</p>"
```

Comment:
Would you consider reverting the changes to Mudrock Snare and its effects to avoid conflicts with https://github.com/foundryvtt/pf2e/pull/16907?